packages/react-admin-core/src/DirtyHandler.tsx

```diff
@@ -93,10 +93,10 @@ export class DirtyHandler extends React.Component<IProps> {
         );
     }
 
-    private registerBinding(obj: React.Component, binding: IDirtyHandlerApiBinding) {
+    private registerBinding(obj: object, binding: IDirtyHandlerApiBinding) {
```

Comment:
only objects, why?

Reply:
dieses obj wird nur dazu verwendet um unregisterBinding machen zu können - dafür brauchen wir was eindeutiges. Das war vorher die klassen instanz - jetzt wurden aber einige komponenten in function components umgebaut und da gibt es kein this mehr. zB da https://github.com/vivid-planet/react-admin/pull/69/files#diff-f2646e56f39f43200fd2b78a59c5392cR50

derm DirtyHandler ist es egal was das genau ist (ref, klassen-instanz, whatever) - darum `object`.

Reply:
klasseninstanz und refs usw wären aber alle typisiert in react aber wird wahrscheinlich zu kompliziert sein da alles abzubilden XD